openthread/CMakeLists.txt

```
@@ -21,6 +23,36 @@ function(checkOpenthreadVersion)
   endif()
 endfunction()
 
+if(CONFIG_OPENTHREAD_SOURCES)
+  message(DEBUG "Building OT from sources, config file will be generated.")
+
+  set(OPENTHREAD_CONFIG_FILE "${CMAKE_BINARY_DIR}/openthread_lib_configuration.txt")
+
+  openthread_libs_configuration_write(${OPENTHREAD_CONFIG_FILE})
+
+  nrfxlib_calculate_lib_path(lib_path)
+  set(OPENTHREAD_DST_DIR "${ZEPHYR_NRFXLIB_MODULE_DIR}/openthread/${lib_path}/v${CONFIG_OPENTHREAD_THREAD_VERSION}")
```

while we are waiting patiently for CI to work, then you might update this to `${CMAKE_CURRENT_LIST_DIR}`.
We are already inside this repo now, so no need to use the module reference.
```suggestion
  set(OPENTHREAD_DST_DIR "${CMAKE_CURRENT_LIST_DIR}/${lib_path}/v${CONFIG_OPENTHREAD_THREAD_VERSION}")
```

---

Done.

---

Next time, you can just hit the `Commit suggestion`, you have an `openthread` too much.